test/utils/runners.go

```diff
@@ -1353,6 +1353,21 @@ func CreatePodWithPersistentVolume(client clientset.Interface, namespace string,
 		// pvc
 		pvc := claimTemplate.DeepCopy()
 		pvc.Name = pvcName
+
+		// Create PVC first as it's referenced by the PV when the `bindVolume` is true.
+		if err := CreatePersistentVolumeClaimWithRetries(client, namespace, pvc); err != nil {
+			lock.Lock()
+			defer lock.Unlock()
+			createError = fmt.Errorf("error creating PVC: %s", err)
+			return
+		}
+		if _, err := client.CoreV1().PersistentVolumeClaims(namespace).UpdateStatus(context.TODO(), pvc, metav1.UpdateOptions{}); err != nil {
```

Comment:
If moving this function call before the original line 1373 `pvc.Status.Phase = v1.ClaimBound`, the pvc status change is not affected on this function. Do I miss something?

Reply:
Nice catch! updated in the new PR.